app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
+            break
```

Comment:
There is a special function in python3 to get the index of a list element. Here's an example:

```python3
list = ['apple', 'banana', 'cherry', 'date']
index = list.index('cherry')
print(index)  # Output: 2
```